ios/Toodle/Rust/List/Items/Item.swift

```diff
@@ -4,58 +4,87 @@
 
 import Foundation
 
-class Item: RustObject {
-    let raw: OpaquePointer
+class Item {
+    var raw: UnsafePointer<CItem>
 
-    required init(raw: OpaquePointer) {
+    required init(raw: UnsafePointer<CItem>) {
         self.raw = raw
     }
 
-    init() {
-        self.raw = item_new()
+    func intoRaw() -> UnsafePointer<CItem> {
+        return self.raw
     }
 
     deinit {
         item_destroy(raw)
     }
 
-    var id: Int? {
-        return item_get_id(raw)?.pointee
+    var uuid: String? {
+        if let uuid = raw.pointee.uuid {
```

Comment:
Do we still need to do this?

Reply:
Yes, because UUID is potentially nil, for now. This is something we are going to address in a follow up